### ISSUE TITLE
Major parts of Canonical's website lacks social metadata card preview images

### ISSUE THREAD

**Issue by johnlettman** [canonical/canonical.com#1101]
<sup><sub>Advanced apologies for the nitpicking issue.</sub></sup>

## Summary
Canonical currently uses a subset of both [Twitter][cards-twitter] and [OpenGraph][cards-og] social metadata across the site:
```html
    <meta name="twitter:account_id" content="169015850">
    <meta name="twitter:site" content="@canonical">
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://canonical.com/">
    <meta property="og:site_name" content="Canonical">
```

However, the metadata lacks information regarding the image when rendering a social media card linking to the site. As a result, Canonical links appear with an image pulled from background elements or with a "missing image" placeholder.

This may cause a slightly negative perception of the link or downplay its visibility in the social media timeline, especially in the case of adverts to open positions, blog posts, or information about Canonical, its services, and its products.

[cards-twitter]: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/abouts-cards
[cards-og]: https://ogp.me/

## Process
1. Open either the [Twitter tweet composer](https://twitter.com/compose/tweet) or [LinkedIn post inspector](https://www.linkedin.com/post-inspector).
2. Paste a Canonical URL into the composer/inspector.
3. Inspect results.

## Expected result
Links to Canonical should be rendered with contextual previews, whether they are product logos, names of the page next to the Canonical logo, or static compositions in the Canonical theme.

## Suggestion actions
The rendered HTML should include the following tags:
```html
<meta name="twitter:image" content="URL to the image">
<meta property="og:image" content="URL to the image">
```

In my opinion, the homepage should take precedence, with product pages probably following behind. All of these display with a variation of the following background element and appear broken:
![image](https://github.com/canonical/canonical.com/assets/20583294/dfc5b715-37f7-48b7-84dc-6ecaa50ed217)

### Canonical pages
[_LinkedIn:_](https://www.linkedin.com/post-inspector/inspect/https:%2F%2Fcanonical.com%2F)
<img src="https://github.com/canonical/canonical.com/assets/20583294/b5496fb1-7c6b-4c0a-bd80-1afb105d5c7e" width="50%" height="auto" alt="A test of a Canonical webpage in the LinkedIn card inspector demonstrating an irrelevant image chosen for the preview image." />

_Twitter:_
![A test of a Canonical webpage in the Twitter tweet composer showing no preview image selected.](https://github.com/canonical/canonical.com/assets/20583294/1310429e-b455-4b3a-9dd4-35aee44022d0)

### vs. OpenStack pages
[_LinkedIn:_](https://www.linkedin.com/post-inspector/inspect/https:%2F%2Fwww.openstack.org%2Fsoftware%2F)
<img src="https://github.com/canonical/canonical.com/assets/20583294/a6c08c2a-353f-4df5-8af7-22180aeaf9f2" width="50%" height="auto" alt="A test of an OpenStack webpage in the LinkedIn card inspector demonstrating a preview image." />

_Twitter:_
![A test of an OpenStack webpage in the Twitter tweet composer demonstrating a preview image.](https://github.com/canonical/canonical.com/assets/20583294/8f940992-7438-482d-9a76-1e0c23a6e484)

## Other views of the problem
<img src="https://github.com/canonical/canonical.com/assets/20583294/2cde97cb-ecf6-4ea8-814b-e94d2de02f48" width="50%" height="auto" alt="'canonical.com' in the LinkedIn post composer." />
<img src="https://github.com/canonical/canonical.com/assets/20583294/497cb25f-10b4-48e1-8df2-7b67654a054e" width="50%" height="auto" alt="'canonical.com/careers' in the LinkedIn post composer." />
<img src="https://github.com/canonical/canonical.com/assets/20583294/97e29a22-6c61-4eba-bae0-14a173183055" width="50%" height="auto" alt="'canonical.com/partners' in the LinkedIn post composer." />
<img src="https://github.com/canonical/canonical.com/assets/20583294/07687c9f-cbd9-41a2-adfc-6aa0866e48b2" width="50%" height="auto" alt="'canonical.com/projects' in the LinkedIn post composer." />

**Comment by johnlettman** [comment]
Also noteworthy is the repetition of `Canonical` in the social metadata for the homepage here:
```html
    <meta name="twitter:title" content="Canonical | Trusted open source for enterprises | Canonical">
    <meta property="og:title" content="Canonical | Trusted open source for enterprises | Canonical">
```

**Comment by johnlettman** [comment]
In the case of the careers page, there should be this image:
<img alt="Canonical careers page social media preview image, two people smiling with a Canonical logomark." src="https://assets.ubuntu.com/v1/c17ef7b4-careers-meta-image-resized.png" width="50%" height="auto" />

However, it appears this templating isn't rendering:
https://github.com/canonical/canonical.com/blob/99034cd0d15c1d70290234dc2dc07c1f23c5438d/templates/base_index.html#L53-L57

<img alt="A screenshot of the LinkedIn post composer showing the desired image for the Canonical careers page not rendering as expected." src="https://user-images.githubusercontent.com/20583294/279485162-497cb25f-10b4-48e1-8df2-7b67654a054e.png" width="50%" height="auto" />

**Comment by carkod** [comment]
I believe this has been fixed?

**Comment by lizzochek** [comment]
@carkod no, we need designers to create banner images, so it was moved to backlog
